Libplanet.Explorer/Queries/TransactionQuery.cs

```diff
@@ -107,6 +117,95 @@ public TransactionQuery()
                 }
             );
 
+            Field<NonNullGraphType<ByteStringType>>(
+                name: "unsignedTransaction",
+                arguments: new QueryArguments(
+                    new QueryArgument<NonNullGraphType<StringGraphType>>
+                    {
+                        Name = "publicKey",
+                        Description = "The hexadecimal string of public key for Transaction.",
+                    },
+                    new QueryArgument<NonNullGraphType<StringGraphType>>
+                    {
+                        Name = "plainValue",
+                        Description = "The hexadecimal string of plain value for Action.",
+                    },
+                    new QueryArgument<LongGraphType>
+                    {
+                        Name = "nonce",
+                        Description = "The nonce for Transaction.",
+                    }
+                ),
+                resolve: context =>
+                {
+                    BlockChain<T> chain = _context.BlockChain;
+                    string plainValueString = context.GetArgument<string>("plainValue");
+                    IValue plainValue = _codec.Decode(ByteUtil.ParseHex(plainValueString));
+                    var action = new T();
+                    action.LoadPlainValue(plainValue);
+
+                    var publicKey = new PublicKey(
+                        ByteUtil.ParseHex(context.GetArgument<string>("publicKey"))
+                    );
+                    Address signer = publicKey.ToAddress();
+                    long nonce = context.GetArgument<long?>("nonce") ??
+                        chain.GetNextTxNonce(signer);
+                    Transaction<T> unsignedTransaction =
+                        Transaction<T>.CreateUnsigned(
+                            nonce,
+                            publicKey,
+                            chain.Genesis.Hash,
+                            new[] { action }
+                        );
+                    return unsignedTransaction.Serialize(false);
+                }
+            );
+
+            Field<NonNullGraphType<StringGraphType>>(
+                name: "attachSignature",
+                deprecationReason: "Use signTransaction",
```

Comment:
It seems this code is from NineChronicles.Headless but the explorer seems not to have the `signTransaction` GraphQL field, right?

Reply:
Right, it would be better copying `signTransaction` instead of it haha.

Reply:
But... well, `attachSignature` may be a better name than `signTranscation` because `signTransaction` doesn't perform sign, but perform attach the given signature. 🤔 

@moreal @planetarium/libplanet What do you think about it?

Reply:
I’m also +1 on `attatchSignature` for better explanation for following code, but if I had to name it I’d call it `bindSignature` for shorter naming.